cmd/osbuild-worker/jobimpl-ostree-resolve.go

```diff
@@ -48,11 +49,31 @@ func (impl *OSTreeResolveJobImpl) Run(job worker.Job) error {
 		Specs: make([]worker.OSTreeResolveResultSpec, len(args.Specs)),
 	}
 
+	config, err := parseConfig(configFile)
+	if err != nil {
+		logrus.Fatalf("Could not load config file '%s': %v", configFile, err)
+	}
+
+	logrus.Info("Composer configuration:")
+	encoder := toml.NewEncoder(logrus.StandardLogger().WriterLevel(logrus.InfoLevel))
+	err = encoder.Encode(&config)
+	if err != nil {
+		logrus.Fatalf("Could not print config: %v", err)
+	}
+
```

Comment:
I am adding configuration parsing for ostree resolve job implementation, I hope it is okay.

The MTLS config from osbuilder is shared for ostree resolving. Will it work this way?